application/dash_application/views/preqc_rna.py

```diff
@@ -195,11 +195,11 @@ def generate_total_reads(df, colour_by, shape_by, cutoff):
 
 def generate_unique_reads(df, colour_by, shape_by):
     return generate(
-        "Unique Reads (Passed Filter)",
+        "Unique Passed Filter Reads (%)",
```

Comment:
In graphs below you use `% X`, whereas here and in a couple of other graphs you use `X (%)`. Can this be standardized?

Reply:
Done